pyiges/iges.py

```diff
@@ -212,9 +212,6 @@ def _return_type(self, iges_type, to_vtk=False, merge=False, **kwargs):
 
         return items
 
-    def __getitem__(self, indices):
-        return self._entities[indices]
-
```

Comment:
I'm confused why this is removed

Reply:
@banesullivan As stated above, this definition of `__getitem__` is overridden (by another definition ca. 100 lines below), would You agree?

Reply:
Ah, sorry, missed that. Thanks for the clarification